src/components/TranslationBanner.tsx

```diff
@@ -133,49 +35,103 @@ const TranslationBanner: React.FC<IProps> = ({
+          md: "rgba(0, 0, 0, 0.16) 0px 2px 4px 0px",
+        }}
+        borderRadius="2px"
+      >
+        <Flex
+          flexDirection="column"
+          alignItems={isPageRightToLeft ? "flex-end" : "flex-start"}
+          m="1rem"
+          mt={{ sm: "2.5rem" }}
+        >
+          <Flex
+            align={{ base: "flex-start", sm: "center" }}
+            mb="1rem"
+            flexDirection={{ base: "column-reverse", sm: "row" }}
+          >
+            <Heading
+              as="h3"
+              size="1.25rem"
+              fontWeight="700"
+              lineHeight="100%"
+              my="0"
+            >
               <Translation id={headerTextId} />
-            </H3>
-            <StyledEmoji
-              ml={"0.5rem"}
-              size={1.5}
+            </Heading>
+            <Emoji
               text=":globe_showing_asia_australia:"
+              pt="0.5rem"
+              ml="0.5rem"
+              my="auto"
+              mb={{ base: "1rem", sm: "auto" }}
             />
-          </Row>
+          </Flex>
           <p>
             <Translation id={bodyTextId} />
           </p>
-          <ButtonRow>
-            <div>
+          <Flex
+            align={{ base: "flex-start", sm: "center" }}
+            flexDirection={{ base: "column", sm: "row" }}
+          >
+            <Box>
               <ButtonLink to="/contributing/translation-program/">
                 <Translation id="translation-banner-button-translate-page" />
               </ButtonLink>
-            </div>
+            </Box>
             {!isPageContentEnglish && (
-              <div>
-                <SecondaryButtonLink
-                  variant="outline"
+              <Box>
+                <ButtonLink
                   to={originalPagePath}
+                  variant="outline"
+                  ml={{ base: 0, sm: "0.5rem" }}
+                  mt={{ base: "0.5rem", sm: 0 }}
+                  borderColor="#333333"
+                  color="#333333"
                   language="en"
                 >
                   <Translation id="translation-banner-button-see-english" />
-                </SecondaryButtonLink>
-              </div>
+                </ButtonLink>
+              </Box>
             )}
-          </ButtonRow>
-        </BannerContent>
-        <BannerClose
+          </Flex>
+        </Flex>
+        <Box
+          position="absolute"
+          top="0"
+          right={isPageRightToLeft ? "auto" : 0}
+          m="1rem"
           onClick={() => setIsOpen(false)}
-          isPageRightToLeft={isPageRightToLeft}
         >
-          <BannerCloseIcon name="close" />
-        </BannerClose>
-      </StyledBanner>
-    </BannerContainer>
+          <Icon
+            as={MdClose}
+            fill="secondary"
+            name="close"
+            boxSize="1.5em"
+            cursor="pointer"
+            _hover={{
+              fill: "primary",
+            }}
+          />
+        </Box>
```

Comment:
Alternatively, the Chakra `CloseButton` component can be used here, which will reduce the amount of code and provide some aria accessibility.

```tsx
<CloseButton
  position="absolute"
  top="0"
  right={isPageRightToLeft ? "auto" : 0}
  margin="0.5rem" // Modification to keep the button's position because of component's rendered size
  color="secondary"
  _hover={{
    color: "primary",
  }}
  onClick={() => setIsOpen(false)}
/>
```